Smtpapi/Smtpapi/Utils.cs

```diff
@@ -65,5 +66,21 @@ public static int DateTimeToUnixTimestamp(DateTime dateTime)
             var span = (dateTime - new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc));
             return (int)span.TotalSeconds;
         }
+
+       
+        /// <summary>
+        /// Checks if string is current year
+        /// </summary>
+        /// <param name="license-date"></param>
+        /// <returns></returns>
+        public static string TestLicenseFileYear(string src)

```

Comment:
This might be my ignorance on csharp, but I wanted to ask - is it necessary to have this in two places?